README.md

```diff
@@ -38,17 +59,75 @@ and install it via `pip` (preferably in a virtual environment):
 ```
 
 
+## Synopsis
+
+```
+datalad make [-i INPUT] [-o OUTPUT] [-p PARAMETER] [-u] TEMPLATE
+```
+
+By design, to perform the computation `datalad make` creates a temporary git
+worktree. All inputs required for the computation are automatically provisioned
+to this temporary worktree, then the specified computation is performed, and
+finally, all requested outputs are transferred back to the original dataset.
+
+The command is invoked with the following arguments:
+
+**`-i INPUT, --input INPUT`** (optional)
+
+Specification of the input file(s) to be provisioned to a temporary git
+worktree. Paths need to be specified relative to the dataset in which `datalad
+make` is executed.
+
+**`-o OUTPUT, --output OUTPUT`**
+
+Specification of the output file(s) to transfer back to the target dataset after
+the computation. Paths need to be specified relative to the dataset in which
+`datalad make` is executed.
+
+**`-p PARAMETER, --parameter PARAMETER`** (optional)
+
+Parameters for the computation, specified in a key-value format (e.g. `-p
+key=value`).
+
+**`-u`** (optional)
```

Comment:
```suggestion
**`-u, --url-only`** (optional)
```
There seems to be a long form, too. Worth including, if only for consistency.

Reply:
Thanks, added!